config/locales/en.yml

```
@@ -4,6 +4,7 @@ en:
     conferences:
       custom_conference_types:
         city: City conferences
+        global: Global conferences
         parallel: Parallel conferences
```

```suggestion
        city: City acts
        international: International acts
        activiy: Parallel activities
```
Let's change the keys too